core/templates/pages/skill-editor-page/services/question-creation.service.ts

```diff
@@ -146,20 +145,20 @@ angular.module('oppia').factory('QuestionCreationService', [
       }
 
       if (noOfRubricsWithExplanation > 1) {
-        $uibModal.open({
-          templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
-            '/pages/topic-editor-page/modal-templates/' +
-              'select-skill-and-difficulty-modal.template.html'),
-          backdrop: 'static',
-          resolve: {
-            allSkillSummaries: () => allSkillSummaries,
-            countOfSkillsToPrioritize: () => countOfSkillsToPrioritize,
-            currentMode: () => currentMode,
-            linkedSkillsWithDifficulty: () => linkedSkillsWithDifficulty,
-            skillIdToRubricsObject: () => skillIdToRubricsObject
-          },
-          controller: 'QuestionsListSelectSkillAndDifficultyModalController'
-        }).result.then(function(linkedSkillsWithDifficulty) {
+        let modalRef: NgbModalRef = NgbModal.open(
+          QuestionsListSelectSkillAndDifficultyModalComponent, {
+            backdrop: 'static'
+          });
+        modalRef.componentInstance.allSkillSummaries =
+          allSkillSummaries;
```

Comment:
Here and below, please wrap in parenthesis when breaking a line.

Reply:
Done.